### ISSUE TITLE
DockerHub readme is truncated and tag links are broken

### ISSUE THREAD

**Issue by maresb** [mamba-org/micromamba-docker#297]
In [this run](https://github.com/mamba-org/micromamba-docker/actions/runs/4756568622/jobs/8452739103#step:6:47) I see the warning

```
Warning: The README content exceeds DockerHub's limit and has been truncated to 25000 bytes.
```

And indeed on Dockerhub I see the end of the README looks like

![image](https://user-images.githubusercontent.com/15216687/233577820-b9b53ea7-95a6-417c-814c-acb5705b69c0.png)

Testing several links it seems that they are all broken, e.g.: https://hub.docker.com/r/ghcr.io/mamba-org/micromamba:jammy-cuda-11.8.0

For the images available on DockerHub, I imagine that using relative links would save several kilobytes, but I'm not sure if it's enough.

**Comment by wholtz** [comment]
I was aware the README was getting truncated, but I didn't know the links to tags had changed.  It is probably time to shift documentation to [ReadTheDocs.org](https://readthedocs.org/) and drastically scale back the content in `README.md`.

**Comment by wholtz** [comment]
I also just noticed that the tag listing currently duplicates each tag. Each tag occurs twice per line.

**Comment by wholtz** [comment]
Duplicate tags is due to tagging both for dockerhub and for ghcr.

**Comment by maresb** [comment]
Makes sense. Is there an easy way to filter them? BTW, great work with ReadTheDocs!

**Comment by wholtz** [comment]
fixed in #328 #329 #330
